drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -37,6 +37,18 @@ def client_search(input:, **args)
       resolve_clients(search_scope, **args)
     end
 
+    clients_field :client_omni_search, 'Client omnisearch' do |field|
+      field.argument :input, Types::HmisSchema::ClientSearchInput, required: true
+    end
+
+    def client_omni_search(input:, **args)
```

Comment:
I'm getting duplicate results in the omnisearch: (there is only 1 Rita Jane Ackroyd Jr in my system)
<img width="613" alt="Screen Shot 2023-01-31 at 8 32 54 AM" src="https://user-images.githubusercontent.com/5333982/215774515-1faf6ba0-b14d-4f2c-b7a3-f93e49b3c92c.png">